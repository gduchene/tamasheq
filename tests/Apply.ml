let f x = x + 1
let x   = f 1
let y   = f (f (f 1))

let g x y z = x + y + z
let h x y   = g x y 5

let z = h 1 2
