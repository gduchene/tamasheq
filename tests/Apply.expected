Apply_0	=	[6] {
    (no name)	=	<abstr:247>
    (no name)	=	2
    (no name)	=	4
    (no name)	=	<abstr:247>
    (no name)	=	<abstr:247>
    (no name)	=	8
}
